File: csharpcore/StarterBot/Entities/Cell.cs
using Newtonsoft.Json;
using StarterBot.Enums;

namespace StarterBot.Entities
{
    public class Cell
    {
        public int X { get; set; }
        public int Y { get; set; }
        public PlayerType PlayerType { get; set; }

        public Cell()
        {
        }

        public Cell(int x, int y, PlayerType playerType)
        {
            this.X = x;
            this.Y = y;
            this.PlayerType = playerType;
        }

        public int getX()
        {
            return X;
        }

        public int getY()
        {
            return Y;
        }

        public PlayerType getPlayerType()
        {
            return PlayerType;
        }

        public bool isPlayers(PlayerType id)
        {
            return id == PlayerType;
        }
    }
}

File: java/src/main/java/za/co/entelect/challenge/entities/Cell.java
package za.co.entelect.challenge.entities;

import za.co.entelect.challenge.enums.PlayerType;

public class Cell {

    protected int x;
    protected int y;
    protected PlayerType playerType;

    public Cell() {

    }

    public Cell(int x, int y, PlayerType playerType) {
        this.x = x;
        this.y = y;
        this.playerType = playerType;
    }

    public int getX() {
        return x;
    }

    public int getY() {
        return y;
    }

    public PlayerType getPlayerType() {
        return playerType;
    }

    public boolean isPlayers(PlayerType id) {
        return id == playerType;
    }
}
